WordPress/Classes/Models/Blog+Creation.swift

```diff
@@ -0,0 +1,37 @@
+extension Blog {
+
+    /// Creates a blank `Blog` object
+    @objc(createWithAccount:orContext:)
+    static func createBlog(with account: WPAccount?, or context: NSManagedObjectContext) -> Blog {
+        let blog = createBlog(in: account?.managedObjectContext ?? context)
```

Comment:
Another nitpick. The fact that the blog is blank seems relevant. What do you think of including it in the method name?

```swift
static func createBlankBlog(with...
```

Reply:
The same goes for the other methods below.

Reply:
Renamed in https://github.com/wordpress-mobile/WordPress-iOS/pull/19925/commits/c6ecf396468587c8d55776493cf328f15d7a545b